CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - The `Api::getPriorties` renamed into `Api::getPriorities` by [@josevh].
 - Remove trailing slash from endpoint url by [@Procta].
 - Added local cache to getResolutions [@jpastoor].
+- Renamed Api::api() parameter $return_as_json to $return_as_array [@jpastoor].
```

Comment:
The `by` is missing before username (in other places `some text by [@username]`). I've noticed it now, but PRs merged today also have that issue.